src/app/views/change-password/changePasswordConfirmed.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Panel from "../../components/panel/Panel";
+
+const propTypes = {
```

Comment:
This page doesn't actually take any properties so we can remove all the propTypes